README.md

```diff
@@ -271,7 +271,7 @@ private void setupCloserBroadcastReceiver() {
         public void onReceive(Context context, Intent intent) {
             // The feedback form has been closed
             // And following result has been returned                
-            FeedbackResult result = (FeedbackResult) intent.getParcelableArrayExtra(FeedbackResult.INTENT_FEEDBACK_RESULT);
+            final FeedbackResult feedbackResult = intent.getParcelableExtra(FeedbackResult.INTENT_FEEDBACK_RESULT_CAMPAIGN);
```

Comment:
Thank you for spotting this problem :)

Though the change in `INTENT_FEEDBACK_RESULT_CAMPAIGN` is not necessary per se, because it depends on which result the user is expecting, the one from a passive form or the one from a campaign